src/calculateRentalCost.js

```diff
@@ -5,6 +5,17 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  const rentForDay = 40;
+
+  if (days >= 7) {
+    return rentForDay * days - 50;
```

Comment:
Extract the '7' and '50' as named constants to avoid hardcoding values directly in the logic, as per checklist item 2. For example, use `const LONG_TERM_RENTAL = 7;` and `const LONG_TERM_DISCOUNT = 50;`.